packages/wdio-cli/src/setup.js

```diff
@@ -6,43 +6,41 @@ import yarnInstall from 'yarn-install'
 
 import { CONFIG_HELPER_INTRO, CONFIG_HELPER_SUCCESS_MESSAGE, QUESTIONNAIRE } from './config'
 
-export default function setup () {
-    console.log(CONFIG_HELPER_INTRO) // eslint-disable-line no-console
-    inquirer.prompt(QUESTIONNAIRE).then((answers) => {
-        let packagesToInstall = []
-        if (answers.installRunner) {
-            packagesToInstall.push(`@wdio/${answers.runner}-runner`)
-        }
-        if (answers.installFramework) {
-            packagesToInstall.push(`@wdio/${answers.framework}-framework`)
-        }
-        if (answers.installReporter) {
-            packagesToInstall = packagesToInstall.concat(answers.reporters)
-        }
-        if (answers.installServices) {
-            packagesToInstall = packagesToInstall.concat(answers.services)
-        }
+export default async function setup (exit = true) {
+    try {
+        console.log(CONFIG_HELPER_INTRO) // eslint-disable-line no-console
+        const answers = await inquirer.prompt(QUESTIONNAIRE)
+        const packagesToInstall = [
```

Comment:
Hey, I'd like to have chromedriver package installed if wdio-chromedriver-service exists in services. 

Not sure if I'm doing things right, if so, is it possible to support it, please? 

https://github.com/webdriverio/webdriverio/pull/4014/files#diff-67c78548af6da581e175d98914982e23

Reply:
Something like this you mean?

```
if (answers.services.includes('wdio-chromedriver-service') {
  packagesToInstall.push('chromedriver')
}
```

But, wouldn't it make more sense to make `chromedriver` a dependency of `wdio-chromedriver-service` that way it's installed when the servie is installed?